addStudy.py

```diff
@@ -0,0 +1,101 @@
+# addStudy.py
+
+#!/usr/bin/python3
+
+# python addStudy.py -c rwa_configuration.yml -l
+# python addStudy.py -c rwa_configuration.yml -a "studyName"
+
+# addStudy.py
+#
+# Adding new studies to the database
+
+# Import our libraries
+import os
+import sys
+import argparse
+import yaml
+import psycopg2
+from include.calibrationLib import *
+
+# List all studies
+def get_studies(configuration, studName):
```

Comment:
Generally it's better to have the function only receive what it needs as a parameter (e.g., the configuration string) as opposed to the entire configuration object.